### PR TITLE
Fixes #1208 - add ETS docs

### LINKED ISSUE #1208: Question: Where to find the schema file for ETS ?
_From @LaurentDardenne on March 25, 2017 15:47_

On this web [page](https://msdn.microsoft.com/en-us/library/ms714674(v=vs.85).aspx) (_Extended Type System (ETS)_) the quoted links are broken :

> For more information about ETS and how Windows PowerShell uses objects, see Windows PowerShell Object Concepts [ps] and Windows PowerShell Extended Type System (ETS) [ps]. 

And this web [page](https://msdn.microsoft.com/en-us/library/dd878306(v=vs.85).aspx) is incomplete.

No details on the _IsHidden_ attribut  : 
```xml 
   <ScriptProperty IsHidden='true'>
```
No example on :
```xml
   <TypeConverter>
```
Or about the specific name '_PropertySerializationSet_' into a propertySet :
```xml
          <PropertySet>
            <Name>PropertySerializationSet</Name>
            <ReferencedProperties>
             ...
```

_Copied from original issue: PowerShell/PowerShell#3422_

### PR DESCRIPTION
**Pull request by sdwheeler**
# PR Summary

Fixes [AB#1098052](https://dev.azure.com/mseng/677da0fb-b067-4f77-b89b-f32c12bb8617/_workitems/edit/1098052) - Fixes #1208 - add ETS docs

## PR Context

Select the area of the Table of Contents containing the documents being changed.

**Conceptual content**
- [ ] Overview and Install
- [ ] Learning PowerShell
  - [ ] PowerShell 101
  - [ ] Deep dives
  - [ ] Remoting
- [ ] Release notes (What's New)
- [ ] Windows PowerShell
  - WMF, ISE, release notes, etc.
- [ ] DSC articles
- [ ] Community resources
- [ ] Sample scripts
- [ ] Gallery articles
- [ ] Scripting and development
  - [x] Legacy SDK

**Cmdlet reference & about_ topics**
- [ ] Version 7.1 preview content
- [ ] Version 7.0 content
- [ ] Version 6 content
- [ ] Version 5.1 content

## PR Checklist

- [x] I have read the [contributors guide][contrib] and followed the style and process guidelines
- [x] PR has a meaningful title
- [x] PR is targeted at the _staging_ branch
- [x] All relevant versions updated
- [x] Includes content related to issues and PRs - see [Closing issues using keywords][key].
- [x] This PR is ready to merge and is not **Work in Progress**
  - If the PR is work in progress, please add the prefix `WIP:` or `[WIP]` to the beginning of the
    title and remove the prefix when the PR is ready.

[contrib]: https://docs.microsoft.com/powershell/scripting/community/contributing/overview
[key]: https://help.github.com/en/articles/closing-issues-using-keywords

### CODE REVIEW

File: reference/docs-conceptual/developer/ets/membersets.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+- **Members** property: Gets all the members of the member set.
+- **MemberType** property: Gets a MemberSet enumeration constant that defines this set as a member
+  set.
+- **Methods** property: Gets the methods included in the member set.
+- **Properties** property: Gets the properties included in the member set.
+- **TypeNameOfValue** property: Gets a MemberSet enumeration constant that defines this set as a
+  member set.
+- **Value** property: Gets the PSMemberSet object.
+- `PSMemberSet.Copy` method: Makes an exact copy of the PSMemberSet object.
+- `PSMemberSet.ToString` method: Converts the PSMemberSet object to a string.
```
Comment (DCtheGeek) [inline reference/docs-conceptual/developer/ets/membersets.md:55]: Formatting on types and objects.

File: reference/docs-conceptual/developer/ets/typeconverters.md [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+
+## Standard Windows PowerShell Language Converters
+
+These standard conversions are checked before any custom conversions and cannot be overridden. The
+following table lists the type conversions performed by PowerShell when the
+`ConvertTo(System.Object,System.Type)` method is called. Be aware that references to the
+**valueToConvert** and **resultType** parameters refer to parameters of the
+`ConvertTo(System.Object,System.Type)` method.
+
+| From (valueToConvert) |  To (resultType)  |                                                                               Returns                                                                               |
```
Comment (DCtheGeek) [inline reference/docs-conceptual/developer/ets/typeconverters.md:22]: In this table, does PSObject need bolding in the type column?

File: reference/docs-conceptual/developer/ets/typeconverters.md [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+methods for converting an object to a specific type or from a specific type. If the
+**valueToConvert** parameter contains an object that has a PowerShell Type converter associated with
+it, PowerShell calls the
+`PSTypeConverter.ConvertTo(System.Object,System.Type,System.IFormatProvider,System.Boolean)` method
+of the associated converter to convert the object to the type specified by the **resultType**
+parameter. If the **resultType** parameter references a type that has a PowerShell type converter
+associated with it, PowerShell calls the
+`PSTypeConverter.ConvertFrom(System.Object,System.Type,System.IFormatProvider,System.Boolean)`
+method of the associated converter to convert the object from the type specified by the
+res****ultType parameter.
```
Comment (DCtheGeek) [inline reference/docs-conceptual/developer/ets/typeconverters.md:76]: What is that supposed to be, `res****ultType`?

**Comment by sdwheeler** [comment]
Retriggering the build

File: reference/docs-conceptual/developer/ets/membersets.md (hunk 1 above)
Comment (sdwheeler) [reply reference/docs-conceptual/developer/ets/membersets.md:55]: fixed

File: reference/docs-conceptual/developer/ets/typeconverters.md (hunk 2 above)
Comment (sdwheeler) [reply reference/docs-conceptual/developer/ets/typeconverters.md:22]: Fixed

File: reference/docs-conceptual/developer/ets/typeconverters.md (hunk 3 above)
Comment (sdwheeler) [reply reference/docs-conceptual/developer/ets/typeconverters.md:76]: fixed

**Comment by sdwheeler** [comment]
@DCtheGeek LTGM #sign-off

**Pull request merged**